### PR TITLE
Setup PillarAnalysis page

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

File: src/components/general/FullPageHeader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { _cs } from '@togglecorp/fujs';
+import Icon from '#rscg/Icon';
+import Heading from '#dui/Heading';
+import _ts from '#ts';
```
Comment (subinasr) [inline src/components/general/FullPageHeader/index.tsx]: If _ts is not used, maybe it's better to remove this.

File: src/views/PillarAnalysis/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { _cs } from '@togglecorp/fujs';
```
Comment (subinasr) [inline src/views/PillarAnalysis/index.tsx]: _cs is not used.

**Review by subinasr — APPROVED** [review]

File: src/components/general/FullPageHeader/index.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                >
+                    {activeProject?.title}
+                </Heading>
+            </div>
+            <div className={_cs(styles.content, contentClassName)}>
+                {children}
+            </div>
+            <div className={_cs(styles.actions, actionsClassName)}>
+                {actions}
+            </div>
```
Comment (frozenhelium) [inline src/components/general/FullPageHeader/index.tsx]: We have `ElementsFragment` component for this layout
```jsx
<div className={_cs(styles.fullPageHeader, className)}>
    <ElementsFragment
        icons={...}
        actions={...}
    >
        {children}
    </ElementsFragment>
</div>
```

File: src/components/ui/BackLink/index.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+function BackLink(props: Props) {
+    const {
+        className,
+        isFirstPage,
+        defaultLink,
+        children,
+        ...otherProps
+    } = props;
+
+    const goBack = useCallback((e) => {
```
Comment (frozenhelium) [inline src/components/ui/BackLink/index.tsx]: use consistent function naming
Suggestion: `handleClick` or `handleLinkClick`

File: src/constants/styleProperties.js [hunk 5]
```diff
@@ -129,11 +129,14 @@ const styleProperties = {
 
         heightActionBar: '42px',
 
+        duiHeightNavbarIcon: '48px',
```
Comment (frozenhelium) [inline src/constants/styleProperties.js]: use relative unit (`rem`)

**Review by frozenhelium — CHANGES_REQUESTED** [review]

File: src/views/PillarAnalysis/index.tsx [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+}
+
+function PillarAnalysis(props: PageProps) {
+    const {
+        pillarId,
+        analysisId,
+        projectId,
+    } = props;
+
+    const [value, setValue] = useState<string | undefined>('');
```
Comment (tnagorra) [inline src/views/PillarAnalysis/index.tsx]:
```suggestion
    const [value, setValue] = useState<string | undefined>();
```

**Review by tnagorra — APPROVED** [review]

**Pull request merged**